db/migrate/058_name_unnamed_redactions.rb

```diff
@@ -0,0 +1,19 @@
+class NameUnnamedRedactions < ActiveRecord::Migration[5.0]
+  def self.up
+  	# after titles are fixed we change the column so to not be null
+    change_column_null "redactions", "title", false
```

Comment:
I think you might need to fill in the titles first, and then change the column constraint afterwards.